telemetry_package/main.py

```diff
@@ -1,33 +1,145 @@
-import sys
+import json
 import rclpy
-from rclpy.node import Node
+import socket
+import os
+import re
 
+from rcl_interfaces.msg import Log
 from common_package_py.common_node import CommonNode
+from rclpy.qos import QoSProfile, QoSReliabilityPolicy, QoSHistoryPolicy, QoSDurabilityPolicy, QoSLivelinessPolicy
 
+DEFAULT_UNIX_SOCKET_PATH = "/tmp/thi_drone"
 
 class TelemetryNode(CommonNode):
-    """A class that represents a telemetry node.
-    """
 
-    def __init__(self, id: str):
-        """Creates a new TelemetryNode.
+    def __init__(self):
+        super().__init__('telemetry_node')
+        qos_profile = QoSProfile( 
+            history=QoSHistoryPolicy.KEEP_LAST,
+            depth=1,
+            reliability=QoSReliabilityPolicy.BEST_EFFORT,
+            durability=QoSDurabilityPolicy.VOLATILE, 
+            liveliness = QoSLivelinessPolicy.AUTOMATIC
+        )
```

Comment:
@J0las could you have a second look at this? 
Looks about right to me.

Reply:
Increase the depth of the History. On the receiving side this would create a transparent buffer if there is a burst of messages.